packages/core/admin/admin/src/content-manager/utils/schema.js

```diff
@@ -207,7 +207,11 @@ const createYupSchema = (
 const createYupSchemaAttribute = (type, validations, options) => {
   let schema = yup.mixed();
 
-  if (['string', 'uid', 'text', 'richtext', 'email', 'password', 'enumeration'].includes(type)) {
+  if (
+    ['string', 'uid', 'text', 'richtext', 'blocks', 'email', 'password', 'enumeration'].includes(
```

Comment:
Yes I understood that after DSM discussion and reverted it. I am also removing max and min length validation as it will break the app otherwise. and adding validation for required attribute only.

Reply:
Thanks Madhuri, should have pinged you when I updated the backend similarly.